src/MountedTraversal.js

```diff
@@ -50,6 +50,9 @@ export function instHasClassName(inst, className) {
     return false;
   }
   let classes = findDOMNode(inst).className || '';
+  if (typeof classes !== 'string') {
+    return false;
```

Comment:
(Note that `node.classList` could be an empty string, so checking it for truthiness is not sufficient - also, rather than using string operations, why not the equivalent of `.split(/\s*/g).includes(className.trim())`?)